src/components/language-switch/language-switch.tsx

```diff
@@ -3,8 +3,13 @@ import React from 'react';
 import styled from 'styled-components';
 import { Link } from '../links/links';
 
+const UnstyledListItem = styled.li`
+  list-style-type: none;
+  display: inline;
+`;
+
 const StyledLanguageLink = styled(Link)<{ selected?: boolean }>`
-  margin: 1px 0px 1px 12px;
+  margin: 1px 0 1px 12px;
```

Comment:
auto change or why did you do this? 0px is valid (as well it's the one without any unit).

Reply:
IntelliJ always shows this as warning, that's why I remove it when I see it 😄 

<img width="483" alt="Screenshot 2021-09-27 at 12 08 20" src="https://user-images.githubusercontent.com/7814926/134889238-30033727-ac9f-4995-a24e-e5fda3dd25b6.png">